packages/react/src/domain-config/Config.css

```diff
@@ -27,4 +29,64 @@
     color: #000;
     background-color: #e9e9e9;
     border-color: #e9e9e9;
+} */
+
+.user-info input {
+    color: #ffffff;
+    background-color: #3A3C50 !important;
+    border-style: none;
+}
+
+.user-info input:focus {
+    color: #ffffff;
+  
+}
+
+.user-info-text-muted {
+    color: #8f8f93 !important;
+}
+
+.user-info-text-highlight {
+    color: #ffffff !important;
```

Comment:
Why so many !importants?

Reply:
Not sure if they are necessary but the complete style needs to be refactored in a separate issue.